apps/root/src/services/accountService.ts

```diff
@@ -341,6 +342,12 @@ export default class AccountService extends EventsManager<AccountServiceData> {
         signature: storedSignature,
       };
 
+      const config = accounts[0].config;
+
+      if (config) {
+        this.web3Service.onUpdateConfig(config);
+      }
```

Comment:
What about current config? We would only be storing new changes applied to the store?